shared/constants/push.js

```diff
@@ -19,6 +19,11 @@ export type PushNotification = {
       convID?: string,
       type?: string,
       username?: string,
+      c?: string,
```

Comment:
Hah, it's part of the payload, which is limited to 2048 bytes, so I wanted to make it as small as possible.